Finjector.Web/ClientApp/src/components/Details/PpmDetails.tsx

```diff
@@ -19,21 +31,35 @@ const PpmDetails: React.FC<PpmDetailsProps> = ({ ppmProjectManager }) => {
       </div>
       <div className="row">
         <div className="col-3 coa-info-title">
-          <h4>Project Name</h4>
+          <h4>Project Type</h4>
         </div>
-        <div className="col coa-details-info-right"></div>
+        <div className="col coa-details-info-right">{projectTypeName}</div>
       </div>
       <div className="row">
         <div className="col-3 coa-info-title">
           <h4>Project Start Date</h4>
         </div>
-        <div className="col coa-details-info-right"></div>
+        <div className="col coa-details-info-right">{projectStartDate}</div>
```

Comment:
will these always be already formatted?

Reply:
It is currently just a string from the API